.sample.env

```diff
@@ -2,4 +2,5 @@ MANDRILL_KEY=GetFromMandrillSettings
 OUTBOUND_EMAIL_DOMAIN=staging.constable.io
 INBOUND_EMAIL_DOMAIN=inbound.staging.constable.io
 SLACK_WEBHOOK_URL=https://fakeslack/webhook
-HUB_API_TOKEN=getFromHubIfNeeded
+CLIENT_ID=GetFromGithub
+CLIENT_SECRET=GetFromGithub
```

Comment:
Where in Github would one find the correct CLIENT_ID? I'm guessing there is a shared resource that only Thoughtbot individuals can access?